### PR TITLE
Remove param translation and shorten param descriptions to safe some flash

### PR DESCRIPTION
**Pull request by sfuhrer**
We're extremely low on flash.

- Assuming that users don't jump one release, we can now delete all the param translations we did up to 1.14 (and now keep adding them again before the next release). 

- I've shortened parameter descriptions (FW, VTOL) a bit

This saves about 2200 Bytes.

### CODE REVIEW

File: src/modules/ekf2/ekf2_params.c [hunk 1]
```diff
@@ -1082,11 +1082,10 @@ PARAM_DEFINE_FLOAT(EKF2_EV_POS_Z, 0.0f);
 /**
 * Airspeed fusion threshold.
 *
-* A value of zero will deactivate airspeed fusion. Any other positive
-* value will determine the minimum airspeed which will still be fused. Set to about 90% of the vehicles stall speed.
-* Both airspeed fusion and sideslip fusion must be active for the EKF to continue navigating after loss of GPS.
-* Use EKF2_FUSE_BETA to activate sideslip fusion.
-* Note: side slip fusion is currently not supported for tailsitters.
+* Airspeed data is fused for wind estimation if above this threshold.
```
[the remark this answers is not in the document]
Comment (KonradRudin) [reply src/modules/ekf2/ekf2_params.c:1085]: It is fine be me actually. its primarily used for wind estimate anyways

File: src/modules/airspeed_selector/airspeed_selector_params.c [hunk 2]
```diff
@@ -1,6 +1,6 @@
 
 /**
- * Airspeed Selector: Wind estimator wind process noise noise spectral density
+ * Wind estimator wind process noise noise spectral density
```
[the remark this answers is not in the document]
Comment (sfuhrer) [reply src/modules/airspeed_selector/airspeed_selector_params.c]: I've amended the corresponding commit.

File: src/modules/ekf2/ekf2_params.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (sfuhrer) [reply src/modules/ekf2/ekf2_params.c:1085]: @bresch does the GPS denied logic check for any of these params? Or for wind validity? 
Or was it just about that GPS denied navigation only really works with a good wind estimate?

**Review by tstastny — APPROVED** [review]

File: src/modules/ekf2/ekf2_params.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (bresch) [reply src/modules/ekf2/ekf2_params.c:1085]: There is no explicit "GPS denied logic", it's just that when there is no other source of vel/pos aiding, the drone needs to currently have a good wind estimate and airspeed fusion active. I think we don't need to mention all the details of GNSS denied navigation in that parameter description.

**Review by bresch — APPROVED** [review]

**Pull request merged**